### ISSUE TITLE
ui5-input does not have aria-label in the input tag

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents#1693]
**Describe the bug**
ui5-input should show aria-label inside the "input" tag e.g. if could use the value of placeholder if present in the <ui5-input .. placeholder="xyz"> or value if present.

**Screenshots**
![Screen Shot 2020-05-27 at 3 57 21 PM](https://user-images.githubusercontent.com/65705637/83080611-901fc980-a033-11ea-854e-98dcccd53b89.png)

**Comment by sleric1024** [comment]
I do also met this issue. 
Form elements should have a visible label while this control does not support this.
